.github/issue-template.md

```diff
@@ -0,0 +1,9 @@
+<!-- Thank you for your contribution. Before you submit the issue:
+- Search open and closed issues for duplicates.
+- Read the [Contributing guidelines](../contributing.md).
+- Follow the [Code of Conduct](../CODE_OF_CONDUCT.md).
```

Comment:
Not about such links here. Maybe we should skip the links?

Reply:
Why?

Reply:
They are not clickable anyway in this form 🤔

Reply:
Ah it is comment. I see what you mean. I'd leave it as is anyway. Maybe someone will pay attention.